### PR TITLE
D2q9 inc

### PR DESCRIPTION
**Pull request by mdzik**
**Proposal**
"Incopressible" d2q9-mrt,
Feq=wi (rho + rho0*(u^2 + etc)

For low Re and Ma doesn’t seem to improve anything
Stored drho instead of rho -> better accuracy for low Ma

### CODE REVIEW

File: src/d2q9_inc/Dynamics.R [hunk 1]
```diff
@@ -0,0 +1,50 @@
+# Density - table of variables of LB Node to stream
```
Comment (llaniewski) [inline src/d2q9_inc/Dynamics.R]: I know that we don't have such a thing in other models (to be filled), but it's good time to start:

Add something like "# Maintainer: Michal Dzikowski @mdzik" So we know who's model is it (as we really get lot of models now.

We should include this in #34 #36

File: src/d2q9_inc/conf.mk [hunk 2]
```diff
@@ -0,0 +1,2 @@
+ADJOINT=0
+TEST=TRUE
```
Comment (llaniewski) [inline src/d2q9_inc/conf.mk]: I think we can exclude this from tests - I'm planning to reduce the number of tested models (or make some different policy for testing) in near future.

**Comment by llaniewski** [comment]
Is this still alive, or is this obsolete wrt #40 ?

**Comment by mdzik** [comment]
Different, aims to be "better" for low-low-speed calculations, rho - 1 trick etc. Nothing to do with #40 
It's working, but I can't see any significant difference (Ladd says, that rho-1 trick is rather important for ~creping flows, even in double precision)
Works & mergable, but probably to extend

**Pull request merged**